src/lib/plugin-helpers.ts

```diff
@@ -81,3 +81,33 @@ type PluginNamespacePath<T extends PluginNamespacePath = "/"> =
 
 /** @var the requested active plugin name (see `src/plugins` for available plugins) */
 export const ACTIVE_PLUGIN = process.env.ACTIVE_PLUGIN;
+
+/**
+ * Returns the active plugins list based on the `ACTIVE_PLUGIN` environment variable.
+ *
+ * The `ACTIVE_PLUGIN` environment variable is a comma-separated list of plugin
+ * names. The function returns the plugins that are included in the list.
+ *
+ * @param plugins is a list of available plugins
+ * @returns the active plugins
+ */
+export function getActivePlugins<T extends { ownedName: string }>(plugins: readonly T[]): T[] {
+  const pluginsToActivateByOwnedName = ACTIVE_PLUGIN
```

Comment:
let's rename this to ACTIVE_PLUGINS now that it's a plural array, and make sure to update .env.local.example as well